database/factories/FlightFactory.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\City;
+use App\Models\Airline;
+use App\Models\Flight;
+use Illuminate\Support\Carbon;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\Flight>
+ */
+class FlightFactory extends Factory
+{
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string, mixed>
+     */
+    public function definition(): array
+    {
+        $departure = now()->addDays(rand(0, 7));
```

Comment:
Carbon instances could be useful for you here. Take a closer look https://carbon.nesbot.com/docs/ It will make you code cleaner and will keep you from using copy